SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -39,7 +43,51 @@ void USpatialGDKEditorSettings::PostInitProperties()
 
 	ULevelEditorPlaySettings* PlayInSettings = GetMutableDefault<ULevelEditorPlaySettings>();
 	PlayInSettings->SetDeleteDynamicEntities(bDeleteDynamicEntities);
-
 	PlayInSettings->PostEditChange();
 	PlayInSettings->SaveConfig();
+
+	SafetyCheckSpatialOSDirectoryPaths();
+}
+
+void USpatialGDKEditorSettings::SafetyCheckSpatialOSDirectoryPaths()
+{
+	const FString path = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial/")));
+
+	FText text = FText::FromString(TEXT("The following directory paths are invalid in SpatialOS GDK for Unreal - Editor Settings:\n\n"));
```

Comment:
Rename this variable to something more suitable, and perhaps keep as an FString until it's ready to be passed to the message dialog.

Reply:
Changed to FString